cross/librsync/Makefile

```diff
@@ -12,6 +12,6 @@ COMMENT  = librsync is a free software library that implements the rsync remote-
 LICENSE  =
 
 GNU_CONFIGURE = 1
-CONFIGURE_ARGS = --disable-static --enable-shared
```

Comment:
Personally, I'd prefer to have `librsync` shared. So, instead of doing this, the alternative is to keep this line, keep the PLIST, add `DEPENDS=cross/librsync` to the spk's Makefile, and remove `COPY_TARGET = nop` from there.